changelog.txt

```diff
@@ -1,3 +1,11 @@
+Version 0.0.5
+-------------
+Adding support for flight and slice parameters to authorization request.
+Updates for supporting authority aliasing
+Added new grant_type: refresh_token
+Updated authorization request base builder to include all MSAL public API properties
+Change from SignedJWT to JWT to support v1
+
```

Comment:
Notes to myself here:

- Internal class renaming to better support IntelliSense
- Access token expiry calculation now supports ext_expires_on